app/formSchema/schema.ts

```diff
@@ -22,10 +22,7 @@ import {
   templateUploads,
 } from './pages';
 
-const useSchema = (formData) => {
-  const isEvidenceOfConnectivityRequired =
-    formData?.templateUploads?.supportingConnectivityEvidence;
-
+const useSchema = () => {
```

Comment:
can schema be an exported constant? That will avoid recreating a new schema object all the time and triggering unnecessary re-renders